src/flat_list.jsx

```diff
@@ -1,71 +1,154 @@
 import React from "react";
 import isEqual from "lodash.isequal";
 
-const timeoutMilliSec = 10;
+const timeoutMilliSec = 0;
 
 const itemChecker = (prevProps, nextProps) => {
   return isEqual(prevProps, nextProps);
 };
 
-const ItemRenderer = React.memo(({ item, renderItem }) => renderItem(item), itemChecker);
+class ItemRenderer extends React.Component {
+  shouldComponentUpdate() {
+    return !itemChecker();
+  }
+  render() {
+    const { item, renderItem } = this.props;
+    return renderItem(item);
+  }
+}
 
 class FlatList extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       items: []
     };
+    this.getLastElementHeight = this.getLastElementHeight.bind(this);
```

Comment:
Same For This .... This Doesn't have to be a class Function

Reply:
No I won't do it. It's not feasible.

Reply:
Ok Sorry